tools/tokenserver/test_scripts.py

```diff
@@ -0,0 +1,223 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I wrote these tests; I figured it would be good to have some end-to-end-ish tests to make sure everything is working as expected